electricitymap/contrib/lib/models/events.py

```diff
@@ -40,9 +41,11 @@ def add_value(
         """
         existing_value: float | None = getattr(self, mode)
         if existing_value is not None:
-            value = 0 if value is None else value
+            value = 0 if value is None or math.isnan(value) else value
             self.__setattr__(mode, existing_value + value)
         else:
+            if value is not None and math.isnan(value):
```

Comment:
Does this work with numpy nans as well ?

Reply:
It does and I added tests for it just to make sure.